scripts/dev_setup.sh

```diff
@@ -428,6 +436,28 @@ function install_cvc4 {
   rm -rf "$TMPFILE"
 }
 
+function install_codegen_deps {
+    if [[ "$PACKAGE_MANAGER" != "apt-get" ]]; then
+      # Not supported.
+      return
+    fi
+    echo "deb http://http.us.debian.org/debian/ buster-backports main" | "${PRE_COMMAND[@]}" tee -a /etc/apt/sources.list
+    echo "deb-src http://http.us.debian.org/debian/ buster-backports main" | "${PRE_COMMAND[@]}" tee -a /etc/apt/sources.list
+    "${PRE_COMMAND[@]}" apt-get update
+    "${PRE_COMMAND[@]}" apt-get upgrade -y
```

Comment:
please don't upgrade

Reply:
this script is rerun at the start of every build job to provide updates in the environment if the script changes, keeping it running fast/minimal output matters.

Reply:
Sure, I suppose I can remove the line `apt-get upgrade -y`.

Reply:
done